tests/bug-6617/test.yaml

```diff
@@ -0,0 +1,17 @@
+pcap: ../filestore-v2.1-forced/suricata-update-pdf.pcap
+
+args:
+- -k none
+
+checks:
+- filter:
+    count: 0
+    match:
+      alert.signature: Filestore toserver
+      alert.signature_id: 1
+
+- filter:
+    count: 1
+    match:
+      alert.signature: Filestore toclient
+      alert.signature_id: 2
```

Comment:
The test should check the stored files (so file info events with the stored property)

Reply:
More precisely, the bug is that files will be stored whatever their direction

Reply:
hmmm I think I understood that but misunderstood `filestore: to_server,flow` keyword. I thought it'll only alert if it is able to store a file in that dir.

Reply:
If I understand correctly, `filestore` keyword is postmatch action